App/Sources/Scenes/Settings/SettingsViewController.swift

```diff
@@ -2,44 +2,94 @@
 //  SettingsViewController.swift
 //  App
 //
-//  Created by Yu Tawata on 2019/05/05.
+//  Created by Yu Tawata on 2019/07/13.
 //
 
 import Foundation
 import UIKit
+import Pipeline
+import BitriseAPI
 import Shared
+import Domain
 
 class SettingsViewController: UIViewController, Instantiatable {
     struct Dependency {
+        let store: StoreProtocol
+        let presenter: SettingsViewPresenterProtocol
     }
 
+    enum SectionKind: Int {
+        case tokens
+        case app
+    }
+
+    enum ItemKind: Hashable {
+        case token(String)
+        case version(String)
+    }
+
+    @IBOutlet weak var contentView: UIView!
     @IBOutlet weak var tableView: UITableView! {
         didSet {
-            tableView.tableFooterView = UIView()
-            tableView.dataSource = self
+            apply(tableView, mainSceneStyle)
             tableView.delegate = self
         }
     }
 
+    private lazy var dataSource = UITableViewDiffableDataSource<SectionKind, ItemKind>(tableView: tableView) { (tableView, indexPath, itemKind) -> UITableViewCell? in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/yuta24/Cimon/blob/1c3e07373f4474a7fd0070b823e8732b727f8267/App/Sources/Scenes/Settings/SettingsViewController.swift#L39"></span>Unused parameter “tableView” in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>SettingsViewController.swift:39</code></td>
    </tr>
  </tbody>
</table>